packages/app/src/cli/services/transform.ts

```diff
@@ -0,0 +1,209 @@
+import * as jscodeshift from 'jscodeshift/src/Runner.js'
```

Comment:
Oddly, this is the location [jscodeshift documents](https://github.com/facebook/jscodeshift#usage-js) for programatic usage. Since the package type is module and this is referencing a file rather than a package, the extension is needed.

Reply:
<img width="508" alt="Screenshot 2023-05-10 at 22 24 09" src="https://github.com/Shopify/cli/assets/6288426/44bcea66-f840-4ea5-a44f-b8716b5e7a4c">

🤔